extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -120,7 +121,9 @@ export class RefreshManager {
     this.adType_ = this.element_.getAttribute('type').toLowerCase();
 
     /** @const @private {?number} */
-    this.refreshInterval_ = this.getPublisherSpecifiedRefreshInterval_();
+    this.refreshIntervalMsecs_ = (refreshIntervalSecs &&
+        this.checkAndSanitizeRefreshInterval_(refreshIntervalSecs)) ||
```

Comment:
You could modify checkAndSanitizeRefreshInterval_ to take undefined thereby removing the check for refreshIntervalSecs here (moves all of the "sanitization" into that function)

Reply:
I'll just add a >= 30000 check so we don't need to call this in the constructor.